packages/odyssey-storybook/src/components/odyssey-mui/Select/Select.stories.tsx

```diff
@@ -527,3 +537,42 @@ export const ControlledPreselectedMultipleSelect: Story = {
     return <Select {...props} value={localValue} onChange={onChange} />;
   },
 };
+
+export const MultipleLanguages: Story = {
+  args: {
+    label: "Select Language",
+    options: optionsLanguages,
+    defaultValue: "",
+  },
+  parameters: {
+    docs: {
+      description: {
+        story:
+          "Showcase the Select component with options for multiple different languages.",
+      },
+    },
+  },
+  play: async ({ canvasElement, step }) => {
+    await step("Select English from the listbox", async () => {
+      const comboBoxElement = canvasElement.querySelector(
+        '[aria-haspopup="listbox"]',
+      );
+      if (comboBoxElement) {
+        await userEvent.click(comboBoxElement);
+        const listboxElement = screen.getByRole("listbox");
+        await expect(listboxElement).toBeInTheDocument();
+        const listItem = listboxElement.children[0];
+        await userEvent.click(listItem);
+        await userEvent.tab();
+        await waitFor(() => {
+          expect(listboxElement).not.toBeInTheDocument();
+        });
```

Comment:
```suggestion
        await waitFor(() => {
          expect(listboxElement).not.toBeVisible();
        });
```

We just care if the user doesn't see it. We don't care if it's gone from the document.

Reply:
For here `toBeInTheDocument` is needed as option items are not visible before any user interaction.